web/styles/compose.css

```diff
@@ -1158,13 +1155,8 @@ textarea.new_message_textarea,
     justify-content: flex-start;
     height: var(--compose-recipient-box-min-height);
 
-    .stream_header_colorblock {
-        margin: 0;
-    }
-
     .dropdown_widget_value {
         flex-grow: 1;
-        overflow: hidden;
```

Comment:
This is causing bottom-left cornor of DM icon to cutt off.
with this property 
![image](https://github.com/zulip/zulip/assets/100200105/8ae73ff4-c887-463b-9540-cd0ec54a7f96)
without this property
![image](https://github.com/zulip/zulip/assets/100200105/3ff31ee1-c2f8-4593-af72-ef1d3d76f7e4)

Reply:
overflow hidden and the ellipsis properties are there for long stream names, ensure you're not breaking their display by removing this property

Reply:
tested
stream picker
![image](https://github.com/zulip/zulip/assets/100200105/f6d899e4-d546-42e2-81fc-99d9228d8f38)

moving message
![image](https://github.com/zulip/zulip/assets/100200105/2ab2a0ee-e996-42dd-bc3e-20011649ee70)